packages/tkg-clusterclass-aws/bundle/config/upstream/overlay.yaml

```diff
@@ -35,4 +35,17 @@ spec:
           status: "False"
           timeout: #@ data.values.MHC_FALSE_STATUS_TIMEOUT
         #@ end
-
+  patches:
```

Comment:
This needs to be added into base.yaml at the beginning of the patches, as there are extraVolume patches in base.yaml. The same is for all cloud providers.